AdminNotificationManagerPlugin.inc.php

```diff
@@ -207,15 +226,16 @@ function _getNotificationSettingsMap() {
 	 * @return array
 	 */
 	function _getContexts() {
-		$contextDao = Application::getContextDAO();
-		$contextIterator = $contextDao->getAvailable();
-		if ($contextIterator && $contextIterator->getCount() > 1) {
-			$contextsById = array();
-			while ($context = $contextIterator->next()) {
-				$contextsById[$context->getId()] = $context->getLocalizedName();
+		$contextIdsObject = new ContextService();
+		$Ids = array();
+		if ($contextIdsObject && $contextIdsObject->getCount() >= 1) {
+			$contextsById = $contextIdsObject->getIds();
+			foreach($contextsById as $context){
+				$arrayValue = $context;
+				$Ids[$arrayValue] = $context;
 			}
 		}
-		return $contextsById;
+		return $Ids;
```

Comment:
Our original return value was an array of context ids as indexes to array values of translated names, e.g. `6 => "My Journal"`.  With the update, this is now context ids as an index to a context id (e.g. `6 => 6`).  Does this have any implications for other plugin function?  If not, can this be a simple array instead (e.g. `array(0 => 6, 1 => 10)`)